Lib/test/test_email/test_generator.py

```diff
@@ -291,6 +292,27 @@ def test_smtputf8_policy(self):
         g.flatten(msg)
         self.assertEqual(s.getvalue(), expected)
 
+    def test_smtp_policy(self):
+        msg = EmailMessage()
+        msg["From"] = Address(addr_spec="foo@bar.com", display_name="Páolo")
+        msg["To"] = Address(addr_spec="bar@foo.com", display_name="Dinsdale")
+        msg["Subject"] = "Nudge nudge, wink, wink"
+        msg.set_content("oh boy, know what I mean, know what I mean?")
+        expected = textwrap.dedent("""\
+            From: =?utf-8?q?P=C3=A1olo?= <foo@bar.com>
+            To: Dinsdale <bar@foo.com>
+            Subject: Nudge nudge, wink, wink
+            Content-Type: text/plain; charset="utf-8"
+            Content-Transfer-Encoding: 7bit
+            MIME-Version: 1.0
+
+            oh boy, know what I mean, know what I mean?
+            """).encode("utf-8").replace(b"\n", b"\r\n")
```

Comment:
Considering to change [`encode("utf-8")`](https://docs.python.org/3/library/stdtypes.html#str.encode) to `encode("ascii")` because the `expected` string contains an email whose `Content-Transfer-Encoding` is 7bit ASCII.

Reply:
I thought that using `encode("ascii")` would ensure that the string is indeed an ASCII-only string, thus supplementing `Content-Transfer-Encoding: 7bit` further and making it explicit.

I’m easy either way, to be honest.